src/cmd/services/m3query/config/config.go

```diff
@@ -65,14 +65,10 @@ type Configuration struct {
 	// Backend is the backend store for query service. We currently support grpc and m3db (default).
 	Backend BackendStorageType `yaml:"backend"`
 
-	// DecompressWorkerPoolCount is the number of decompression worker pools.
-	DecompressWorkerPoolCount int `yaml:"workerPoolCount"`
+	// ReadWorkerPoolSize is the size of the worker pool for read requests.
```

Comment:
more a fyi than anything else: this changes a publicly accessible field. I think it's fine for now considering i don't think we've made a release exposing this in configs and put it in docs at all. But as we add more features, we have to make them in a b/w compatible way to cause the least friction for users.

Reply:
Yeah, agreed. This is not a field that would currently be in use, and is currently undocumented, so I'm ok with being a bit fast and loose with changing it at the moment 😛